apps/codeforafrica/src/components/ArticleCardList/ArticleCardList.js

```diff
@@ -37,15 +37,15 @@ const ArticleCardList = React.forwardRef(function ArticleCardList(props, ref) {
                   display: { xs: "none", sm: "block" },
                   height: "217.64px",
                 }}
-                src={article.featureImage}
+                src={article.image.src}
```

Comment:
@kelvinkipruto can we guarantee that image can never be null or undefined?

Reply:
@koechkevin Yes. The cover image is a required field